gb-cpu/src/bus.rs

```diff
@@ -0,0 +1,8 @@
+pub trait Bus<T> {
+    type Item;
+    type Result;
+    type Data;
+
+    fn get(&self, _: T) -> Self::Item;
```

Comment:
_ is a prefix for a variable not used in a regular function. In a type definition it's a placeholder representing the T type, the name changes depending on the specific implementation

Reply:
It would make easier to understand this function behavior if its parameters are named in its definition. Even if the parameter type is generic we should find a name that can indicate its general behavior.  

In this case, I guess it could be `index` or `position` ?

Reply:
position is great imo!